registration/tests/test_model.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+
+from registration.models import Charity, Venue
+from registration.tests.common import DEFAULT_VENUE_ARGS
+
+
+class TestCharity(TestCase):
+    def setUp(self):
+        self.charity = Charity(name="Reconstructing the Past", donations=5.00)
+        self.charity.save()
+
+    def test_charity_fields(self):
+        self.assertIsInstance(self.charity.name, str)
+        self.assertIsInstance(self.charity.url, str)
+        self.assertIsInstance(self.charity.donations, float)
```

Comment:
I'm not sure how useful testing instance types like on a model is - [also not sure how this test is passing] - the underlying type for donations is a `Decimal`, and the ORM will keep the input type passed into object until an explicit refresh or fetch.

Try adding `self.charity.refresh_from_db()` after it's saved.

Reply:
that did the trick